system-addon/test/unit/lib/SectionsManager.test.js

```diff
@@ -210,23 +210,38 @@ describe("SectionsManager", () => {
     });
   });
   describe("#updateBookmarkMetadata", () => {
-    let rows;
+    let clock;
+
     beforeEach(() => {
-      rows = [{
+      clock = sinon.useFakeTimers();
```

Comment:
What's the clock for?

Reply:
Testing the call arguments on `insert`, one of the arguments is `new Date()`.